lib/griddler/mandrill/adapter.rb

```diff
@@ -44,7 +44,7 @@ def recipients(field, event)
 
       def resolve_bcc(event)
         email = event[:email]
-        if !event[:to].map(&:first).include?(email) && event[:cc] && !event[:cc].map(&:first).include?(email)
+        if !event[:to].map(&:first).include?(email) && (!event[:cc] || !event[:cc].map(&:first).include?(email))
```

Comment:
I just hit this bug myself and it was not trivial to see the issue in this line - it is complicated enough. Could we refactor this condition to be more readable and maintainable?